src/modules/admin/components/projectConfig/ProjectConfigTable.tsx

```diff
@@ -62,9 +65,24 @@ const columns: ColumnDef<AutoExitConfigFieldsFragment>[] = [
         <NotCollectedText variant='body2'>Any Organization</NotCollectedText>
       ),
   },
+  {
+    header: 'Config Options',
+    render: ({ configOptions }) => {
+      if (!configOptions) return;
+      const parsedOptions = JSON.parse(configOptions);
+
+      return Object.keys(parsedOptions).map((key) => {
+        return (
+          <span key={key}>
```

Comment:
this whole thing will need design review of course, but this is a particular place in the code where we may need some thoughts from Ash on how to display the newly generified key/value pairs of config options

Reply:
Agreed. Lets just change the `span` to a `Stack` for now so that multiple attributes would  stack vertically.